vignettes/getting_started.Rmd

```diff
@@ -0,0 +1,194 @@
+---
+title: "Setting up checklist"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Setting up checklist}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Starting an R package from scratch
+
+### Prepare online setup
+
+1. Go to https://github.com and login.
+  Create a new public repository in the INBO organisation.
+  Use the package name as repository name.
+  Enter the repository name and optionally a description.
+  Don't add a `README`, `.gitignore` or `LICENSE`.
+
+### Prepare local setup
+
+1. Use `create_package()`.
+  You'll need to provide the package name, title, description and maintainer.
+  Please read the documentation of `create_package()` prior to running the function.
```

Comment:
As I already stated there: the explanation on how to provide a maintainer, is insufficient, so please provide some more documentation in this function (just an example could explain a lot, I think).  Referring to the function documentation here, while it is poorly documented there, can be frustrating for users.

Reply:
Done in https://github.com/inbo/checklist/pull/38/commits/2a2283f6e02a72f882b6a0656b27c7e548a8a8a8